### PR TITLE
replay(feat): Remove Replay controls from Issue Details page (et al), replace with a CTA

### LINKED ISSUE #39556: Bug: Replay preview can be huge, needs a max-width and possibly min-width
The replay preview in the Issue Details page takes up as much space as it can when the pageloads, but after resizing the window it'll shrink down and stay at the smaller size. This is weird behavior, and it shouldn't be taking up that much space to begin with imo.

We should add a `max-height` and possibly `min-height` into static/app/components/events/eventReplay/replayContent.tsx `<PlayerContainer>`

| Step  | Result |
| ------------- | ------------- |
| Load page - wide  | Image |
| Resize window - narrower | Image |
| Resize window - bigger again | Image |

### LINKED ISSUE #39630: Replay preview on issues detail should be static, show a big purple play icon and no ReplayController

### PR DESCRIPTION
**Pull request by ryan953**
Updates to the Replay Preview widget that appears on the Issue Details, and Performance summary pages.

- Removed the scrubber and play/pause buttons below the replay
   - not all the buttons were working or making sense on this page, for example the 'next breacrumb' button doesn't interact with the breadcrumb table that is rendered just below.
   - Kept the new 'View Full Replay' button in the bottom-right corner, this is a nice Call to Action
- Constrained the height so the preview isn't as huge as possible
   - So we'll see the replay be letterboxed more often
   - The video will shrink if you resizing the window to be really small which is fine. But if you make the window large again the video stays small. Minor nit not addressing today.
- Added the purple circular Play button
    - The whole replay preview is clickable and will deep link you into the replay details page.

| Desc | Img |
| --- | --- |
| Before  | Screen Shot 2022-10-05 at 12 57 14 PM |
| After | Screen Shot 2022-10-05 at 12 57 01 PM | 

Fixes #39630
Fixes #39556

### CODE REVIEW

File: static/app/components/events/eventReplay/replayContent.tsx [hunk 1]
```diff
@@ -57,35 +56,43 @@ function ReplayContent({orgSlug, replaySlug, event}: Props) {
...
+          </CTAOverlayLink>
+
+          <StaticPanel>
+            <ReplayPlayer />
+          </StaticPanel>
+        </FluidHeight>
+
+        <CTAButtonContainer>
+          <Button
+            data-test-id="view-replay-button"
```
Comment (billyvg) [inline static/app/components/events/eventReplay/replayContent.tsx:87]: This is existing code, but `data-test-id` on buttons shouldn't be needed, we should be using `aria-label` selector for all buttons.

**Review by billyvg — APPROVED** [review]
Maybe I'm missing something but don't we want the bigger replay?

**Comment by ryan953** [comment]
> Maybe I'm missing something but don't we want the bigger replay?

The replay size before was unbounded, which made it easy to take over the page. The new max size is super-arbitrary, but with this change there's no play button anymore, it's more of a tease to get people to click through. So think smaller is better as a teaser, and if the user isn't looking at the replay they can scroll past it quicker.

File: static/app/components/events/eventReplay/replayContent.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/components/events/eventReplay/replayContent.tsx:87]: updated, there are two buttons on the page with that label, and pointing to the same href.

**Pull request merged**